environment_setup/setup_software.sh

```diff
@@ -126,6 +127,7 @@ if ! sudo apt-get install bazel-3.7.2 -y ; then
     echo "##############################################################"
     exit 1
 fi
+sudo ln -s /usr/bin/bazel-3.7.2 /usr/bin/bazel
```

Comment:
Added a link between `bazel-3.7.2` to `bazel`. Not sure if I should've added it else where so it only runs for Ubuntu 20 or not. Ended up adding it here to be after Bazel is installed.

Reply:
This seems like it is also recommended in the bazel docs:

https://docs.bazel.build/versions/master/install-ubuntu.html#step-2-install-and-update-bazel

we just gotta make sure that it doesn't break CI and that the script is repeatable (can you run this script again and make sure it doesn't fail on this line)

Reply:
I tried running it again on my machine and the script did not crash, and was able to link them when I unlinked bazel.